### ISSUE TITLE
I noticed, all the codes for the eur version don't work, they can just freeze the game or have no effect

### ISSUE THREAD

**Issue by H0M3BR3W** [ThisisCanada98/DragonballFusionsNTR#2]
i'm on the eur cia version

**Comment by ProjectZ83** [comment]
Hmm... I'm not sure. So all of them don't work?

**Comment by H0M3BR3W** [comment]
@ProjectZ83 in fact i have not tested the streetpass codes but the sections "main character (move 1 and fused move)" don't work, the main character rank also don't work, team a slot 1 the same. and the miscellaneous codes like 999 enregies, all dragons balls, all characters also don't work. i have not tested the max ultra gauge and max ki bc i have another plugin which does it perfectly. all training complete cannot be tested bc i finished myself the training list and secondary characters max level my other plugin does it. i installed this plugin to have all characters but nothing work haha

**Comment by ProjectZ83** [comment]
Maybe the plugins are contradicting each other or it might be your game version. Are you on 2.2.0

**Comment by H0M3BR3W** [comment]
I don't think so, and yeah i'm on 2.2.0, and i use all plugins separately and i switched to the usa version of the game i intalled the usa plugin and it worked fine, so i'm pretty sure that's a common problem people have bc i'm not the first who reports crashes on eur version

**Comment by Artorikus** [comment]
90% of the cheats doesn't works for me too...
